interfaces/openapi-to-go-server/apis.py

```diff
@@ -58,6 +59,19 @@ def filter_operations(self, tags: Set[str]):
         self.data_types = [dt for dt in self.data_types
                            if dt.name in required_data_types]
 
+    def security_scopes(self) -> Set[Tuple[str, operations.Scope]]:
```

Comment:
Might a `Dict` make more sense? 

A set of tuples is a lot like a Dict, though if you don't need to do lookups, the Set is fine.